extensions-core/lookups-cached-global/src/main/java/io/druid/query/lookup/NamespaceLookupExtractorFactory.java

```diff
@@ -125,7 +131,12 @@ public boolean start()
   public boolean close()
   {
     final Lock writeLock = startStopSync.writeLock();
-    writeLock.lock();
```

Comment:
For the factory start/stop nothing explicitly calls interrupts here, but the general "update" method when the coordinator pushes stuff to the historicals does call `manager.updateIfNew`, which calls `start()`. I would like to eventually have the POST handlers be able to kill their own threads appropriately for stuck items. As such I propose retaining the interruptibility of the start and stop locks in the Factory class.